docker-local/docker-compose.yml

```diff
@@ -59,6 +59,23 @@ services:
       - storjsim
       - authservice
       - tracing
+  linksharing:
+    image: golang:1.17.7
+    entrypoint: /scripts/start-linksharing.sh
+    volumes:
+      - localgopath:/go/storjpath
+      - ${GATEWAY_SRC_DIR}:/code
+      - ./start-linksharing.sh:/scripts/start-linksharing.sh
+    ports:
+      - "8001:8001"
```

Comment:
same here re: ports?

Reply:
I've removed this commit from this PR, I shouldn't have stacked them like that.